design/comment-scanning.md

```diff
@@ -0,0 +1,159 @@
+# Polymer Analyzer - Comment Directives & Comment Annotations
+> Author @FredKSchott
+> Status: Draft
+> Last Update: 2017-04-26
+
+## Objective
+- To analyze standalone comments (directives) for instructions.
+- To analyze "attached" comments (annotations) for information/metadata about an existing feature.
+
+
+## Goals
+- To scan & analyze polymer-lint directives so that the linter can properly report & ignore warnings in a document.
+- Create an extensible system for analyzing and surfacing standalone comments in a document.
```

Comment:
I wanted to flesh out this definition to answer the "what if I want to attach information to a feature" question, but I agree that I've probably gone into too much detail at this point. 

Let me see if I can boil this down to just the relevent-to-directives bits.